content/2.0.html.haml

```diff
@@ -130,6 +130,14 @@ title: "Jenkins 2.0"
       !
 
   %dt
+    Out of the box: Secure defaults
+
+  %dd
+    %p
+      Jenkins access is no longer "first come, first served":
```

Comment:
It took me some time to realize you are talking about the first time setup. I'd put it more straightly, like "Jenkins helps you secure itself right after the installation through the setup wizard" or something.

Just my 2 cents.